### PR TITLE
Bfb/analyze all teams in alliance

### PR DESCRIPTION
**Pull request by BrendanBurkhart**

### CODE REVIEW

File: internal/server/logic/events.go [hunk 1]
```diff
@@ -44,7 +45,7 @@ func Analyze(eventKey string, teams []string, schema analysis.Schema, rs report.
 		}
 
 		if len(stats) == 0 {
-			return []TeamAnalysis{}, nil
```
Comment (fharding1) [inline internal/server/logic/events.go]: Why?

File: vendor/vendor.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
 	"comment": "",
+	"heroku": {
```
Comment (fharding1) [inline vendor/vendor.json]: Why?

File: internal/server/logic/events.go (hunk 1 above)
Comment (BrendanBurkhart) [reply internal/server/logic/events.go]: Previously if there wasn't stats available for one of the teams on the alliance it just didn't return any averages for the whole alliance. Now it skips over teams with no data available, but includes stats for the other teams on the alliance.

File: vendor/vendor.json (hunk 2 above)
Comment (BrendanBurkhart) [reply vendor/vendor.json]: That was auto generated, do we even want to keep using govendor?

File: internal/server/logic/events.go (hunk 1 above)
Comment (fharding1) [reply internal/server/logic/events.go]: :+1:

File: vendor/vendor.json (hunk 2 above)
Comment (BrendanBurkhart) [reply vendor/vendor.json]: Also that was previously at the bottom, I have no idea why govendor decided to move it up to here.

Comment (fharding1) [reply vendor/vendor.json]: We probably should... but we don't need the heroku stuff.

**Pull request merged**